pyproject.toml

```diff
@@ -61,15 +59,10 @@ addopts = "-ra"
 filterwarnings = [
   "ignore:.*does not have many workers which may be a bottleneck.*:UserWarning",
   "ignore:.*Found no NVIDIA driver on your system.*:UserWarning",
-  "ignore:the imp module is deprecated in favour of importlib:DeprecationWarning",
   "ignore:GPU available but not used.*:UserWarning",
-  "ignore:`np.bool` is a deprecated alias for the builtin `bool`:DeprecationWarning",
-  "ignore:`np.object` is a deprecated alias for the builtin `object`:DeprecationWarning",
+  "ignore:numpy.ufunc size changed, may indicate binary incompatibility. Expected 216 from C header, got 232 from PyObject:RuntimeWarning",
   "ignore:numpy.ndarray size changed:RuntimeWarning",
-  "ignore:`pytorch_lightning.metrics.*` module has been renamed to `torchmetrics.*` and split off to its own package:DeprecationWarning",
-  "ignore:Using or importing the ABCs from 'collections' instead of from 'collections.abc':DeprecationWarning",
-  "ignore:The `LightningModule.datamodule` property is deprecated in v1.3 and will be removed in v1.5. Access the datamodule through using `self.trainer.datamodule` instead:DeprecationWarning",
-  "ignore:ray.services.get_node_ip_address has been moved to ray.util.get_node_ip_address. 'ray.services' will be removed after Ray 1.4.:DeprecationWarning",
+  "ignore:.*torch.meshgrid.*:UserWarning",
 ]
 markers = [
   "multi_gpu: marks tests that requires multiple gpus to run",
```

Comment:
Is this marker still relevant? I think tune.py was the only code using multiple GPUs.

Reply:
yeah let's get rid of it